Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/TaskListActivity.java

```diff
@@ -0,0 +1,88 @@
+package pg.autyzm.friendly_plans;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.inject.Inject;
+
+import database.entities.TaskTemplate;
+import database.repository.TaskTemplateRepository;
+
+;
+
+/**
+ * Created by Mateusz on 2017-03-08.
+ */
+
+public class TaskListActivity extends AppCompatActivity {
+
+    @Inject
+    TaskTemplateRepository taskTemplateRepository;
+
+    private TaskRecyclerViewAdapter taskListAdapter;
+    private static final String TAG = "TaskListActivity";
+
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+
+        super.onCreate(savedInstanceState);
+        ((App) getApplication()).getRepositoryComponent().inject(this);
```

Comment:
hmm i'm sure that i've done it , i've must check my commits once again and fix it